cli/build.rs

```diff
@@ -338,14 +343,23 @@ fn create_cli_snapshot(snapshot_path: PathBuf, esm_files: Vec<PathBuf>) {
     deno_flash::init::<PermissionsContainer>(false), // No --unstable
   ];
 
+  let mut esm_files = include_js_files_dir!(
+    dir "js",
+    "40_testing.js",
+  );
+  esm_files.push(ExtensionFileSource {
+    specifier: "runtime/js/99_main.js".to_string(),
+    code: deno_runtime::js::SOURCE_CORE_FOR_99_MAIN_JS,
```

Comment:
This seems a little crufty - I believe the resulting specifier will be `internal:deno_cli/runtime/js/99_main.js`. OTOH it's not a big deal.

Reply:
i think its fine; no one would import from it, and i doubt errors will come from it